examples/api/lib/material/slider/slider.1.dart

```diff
@@ -0,0 +1,66 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
Can you point to me where this example is used? I don't see it referred to anywhere in doc.

Reply:
Add in latest commit